configs/nvme-storage-configurator/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
```

Comment:
should we set the `namespace` here?

Reply:
Should we? If we should, which namespace? the one where DaemonSet will run?

Reply:
removed the `namespace.yaml` and added namespace attribute in `kustomisation.yaml`